### DISCUSSION TITLE
[Feature Request] Slug format, like title format

### DISCUSSION THREAD

**Discussion by benface** [craftcms/cms#8624] [Ideas]
Right now, the auto-generated slugs are always based on the entry's title. It would be great if we could customize that, just like we can change the auto-generated title format. We could base the slug on another field, for instance, or on something else entirely. I imagine the "kebabing" of the slug to make it URL-friendly would still apply on top of whatever format we choose, though.

**Comment by davidhellmann** [comment]
Count me in. Need this feature urgent ☺️

**Comment by carlcs** [comment]
When using a title format I’m basically thinking of the title as my slug generator field, it does exactly what you’re describing, compose the slug from other fields.

**Comment by carlcs** [comment]
Just noticed that my comment might be mistaken as a contra argument. I like the FR, it would be good to have slug generating separate from the h1 on edit entry pages and first column in the entries table.

For the frontend I don’t mind to access a custom field instead of the title.
